css/properties/-webkit-max-logical-width.json

```
@@ -20,7 +20,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "≤13.1"
+              "version_added": "≤5.1"
```

```suggestion
              "version_added": "5.1"
```

Verified, similar to `-webkit-logical-height`.